fs_storage/models/fs_storage.py

```diff
@@ -21,6 +21,8 @@
 
 _logger = logging.getLogger(__name__)
 
+LS_NON_EXISTING_FILE = ".NON_EXISTING_FILE"
```

Comment:
Should it be a new config field? Should we add some random characters when using it to ensure it is non-existing on remote FS?